src/Native/Unix/gen-buildsys-clang.sh

```diff
@@ -7,7 +7,7 @@ if [ $# -lt 4 -o $# -gt 6 ]
 then
   echo "Usage..."
   echo "gen-buildsys-clang.sh <path to top level CMakeLists.txt> <ClangMajorVersion> <ClangMinorVersion> <Architecture> [build flavor] [cmakeargs]"
-  echo "Specify the path to the top level CMake file - <corefx>/src/Native"
+  echo "Specify the path to the top level CMake file - <corefx>/src/Native/Unix"
```

Comment:
I want to mimic the behavior we have for windows when building native binaries. Looking at the folder structure we had before it was not clear that by default everything that was not under the windows folder was being used when we were building in unix.